suppress.xml

```diff
@@ -1,5 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-    <!-- Empty file for now. Suppressions may be added later when analysing reported vulnerabilities.
-         See https://jeremylong.github.io/DependencyCheck/general/suppression.html -->
+    <!-- Suppressions may be added as outlined at: https://jeremylong.github.io/DependencyCheck/general/suppression.html -->
+    <suppress until="2023-09-30">
```

Comment:
No mention of this change in the PR description / commit message. Suggest adding it (assuming this change is meant to be here too).